dcos_core_variables.tf

```diff
@@ -523,3 +523,38 @@ variable "dcos_versions_service_url" {
   default     = "https://versions.d2iq.com"
   description = "DC/OS Versions Service allows to identify DC/OS versions"
 }
+
+variable "dcos_calico_network_cidr" {
+  default     = ""
```

Comment:
What do we think about defaulting to something like `172.29.0.0/16`? This is a fairly unused subnet, being at the high end of `172.16.0.0/12` but not at the very end of it. I think having a good default while keeping the ability to override here would make the end user experience better.

Reply:
I second you on making the end-user experience better if possible, but I insist on using mandatory calico subnet configuration, the side effect of using the overlapped networks we had mentioned in https://github.com/dcos/dcos/pull/6244#discussion_r340788009
but we need to draw a balance between making 95+% of our customers happy, and the left ones in anger.
I am happy to discuss this.